docs/reference/modules/task.md

```diff
@@ -0,0 +1,97 @@
+# `@apostrophecms/task`
+
+**Alias:** `apos.task`
+
+This module allows support modules in creating command line tasks. It also provides utilities for generating request objects when one is not available and is needed.
+
+Command line tasks are invoked with the general structure:
+
+```bash
+node app module-name:task-name --arguments
+
+# Example
+node app @apostrophecms/migration:migrate
+node app article:generate --total=20
+```
+
+Apostrophe is fully initialized before a task is run, except that it does not listen for connections. We may access all general Apostrophe features in a task.
+
+## Related documentation
+
+- [Module task configuration](/reference/module-api/module-overview.md#tasks-self)
+
+## Featured methods
+
+The following locales belong to this module and may be useful in project-level code. See the [source code](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/task/index.js) for all methods that belong to this module.
```

Comment:
locales -> methods

Reply:
Because this module has an alias you can invoke these succinctly from another module, for instance: `self.apos.tasks.getReq(...)`